components/status_dropdown/status_dropdown.scss

```diff
@@ -0,0 +1,98 @@
+.custom_status__row {
+    display: flex;
+    font-size: inherit;
+    padding: 0 24px;
+    align-items: center;
+    height: 34px;
+    max-width: 238px;
+    &:hover {
+        background-color: rgba(var(--center-channel-color-rgb), 0.08);
+    }
+
+    &:active {
+        background-color: rgba(var(--button-bg-rgb), 0.08);
+        color: var(--button-bg);
+    }
+}
+.custom_status__icon{
+    margin-right: 10px;
+}
+
+.custom_status__text{
+    overflow: hidden;
+    height: 20px;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.custom-status-suggestion-emoji > span:first-child {
+    background-size: 20px;
+}
+
+.custom_status__clear {
+    display: flex;
+    align-items: center;
+    margin-left: auto;
+    color: rgba(var(--center-channel-color-rgb), 0.52);
+}
+
+.pulsating_dot, .pulsating_dot:before, .pulsating_dot:after {
+    height: 12px;
+    width : 12px;
+    background-color : var(--online-indicator);
+    border-radius : 50%;
+}
+
+.pulsating_dot {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-left: auto;
+    position: relative;
+}
+
+.pulsating_dot:before, .pulsating_dot:after {
+    content:"";
+    display:block;
+    position: absolute;
+    left: 0;
+    top: 0;
+}
+
+.pulsating_dot:after {
+    animation: pulse1 2s ease 0s infinite;
+}
+
+.pulsating_dot:before {
+    animation: pulse2 2s ease 0s infinite;
+}
```

Comment:
could you please simplify the `.pulsating_dot` class with SCSS `Current Selector` as well?